shared/common-adapters/icon.js.flow

```diff
@@ -3,7 +3,7 @@
 import React, {Component} from 'react'
 
 export type Props = {
-  type: 'GPG-export' | 'computer-big' | 'documentation' | 'keybase-update-pause' | 'keybase-update' | 'logo-15' | 'logo-24' | 'logo-128' | 'logo-256' | 'logo-32' | 'logo-64' | 'no-gpg' | 'paper-key' | 'phone-big' | 'phone-oblique' | 'phone-q-r-code-small' | 'phone-q-r-code' | 'phone-text-code-small' | 'phone-text-code' | 'placeholder-avatar' | 'terminal' | 'user-card'
+  type: 'GPG-export' | 'computer-big' | 'documentation' | 'keybase-update-pause' | 'keybase-update' | 'invite-code-m' | 'logo-15' | 'logo-24' | 'logo-128' | 'logo-256' | 'logo-32' | 'logo-64' | 'no-gpg' | 'paper-key' | 'phone-big' | 'phone-oblique' | 'phone-q-r-code-small' | 'phone-q-r-code' | 'phone-text-code-small' | 'phone-text-code' | 'placeholder-avatar' | 'terminal' | 'user-card'
```

Comment:
That's your huge GPG PR, though -- the icons don't even have their own commit inside it -- so I don't see how to include that without messing up the ability to get this diff against master reviewed.

I did use the same icon name (invite-code-m) as you, so the files themselves should merge okay, just leaving a conflict on the Props type.